io.catenax.transmission.transmission_pass/1.0.0/TransmissionPass.ttl

```diff
@@ -0,0 +1,520 @@
+    bamm:characteristic :DocumentsCharacteristic.
+:packagingInstructions a bamm:Property;
+    bamm:preferredName "Packaging instructions"@en;
+    bamm:description "Instructions for safely packaging of the transmission."@en;
+    bamm:characteristic :DocumentsCharacteristic.
+:transportationInstructions a bamm:Property;
+    bamm:preferredName "TransportationInstructions"@en;
+    bamm:description "Instructions for safely transporting of the transmission."@en;
+    bamm:characteristic :DocumentsCharacteristic.
+:safetyMeasures a bamm:Property;
+    bamm:preferredName "Safety measures"@en;
+    bamm:description "Includes all safety relevant documents for e.g. Dismantling."@en;
+    bamm:characteristic :DocumentsCharacteristic;
+    bamm:see <https://eclass.eu/eclass-standard/content-suche/show?tx_eclasssearch_ecsearch%5Bcc2prdat%5D=0173-1%2302-AAO190%23001&tx_eclasssearch_ecsearch%5Bdischarge%5D=0&tx_eclasssearch_ecsearch%5Bid%5D=-1&tx_eclasssearch_ecsearch%5Blanguage%5D=1&tx_eclasssearch_ecsearch%5Boption%5D=pr&tx_eclasssearch_ecsearch%5Bsearchtxt%5D=0173-1%2302-AAO190%23001&tx_eclasssearch_ecsearch%5Bversion%5D=12.0&cHash=ecd65dc92a134cdb1453556e4a1557ca>.
+:DocumentsCharacteristic a bamm-c:List;
+    bamm:preferredName "Documents Characteristic"@en;
+    bamm:description "Documents Characteristic with title and link."@en;
+    bamm:dataType :Documents.
+:Co2Total a bamm-c:Quantifiable;
+    bamm:preferredName "CO2 Total"@en;
+    bamm:description "Characteristic for the total carbon footprint of the transmission, calculated as kg of carbon dioxide equivalent."@en;
+    bamm:dataType xsd:decimal;
+    bamm-c:unit unit:kilogram.
+:RecyclateContent a bamm:Characteristic;
+    bamm:preferredName "Recyclate Content"@en;
+    bamm:description "Recyclate content in the transmission."@en;
+    bamm:dataType :RecyclateContentEntity.
+:StringList a bamm-c:List;
+    bamm:preferredName "String List"@en;
+    bamm:description "A list with the data types string."@en;
+    bamm:dataType xsd:string.
+:ProductDescription a bamm-c:Enumeration;
+    bamm:preferredName "Product Description"@en;
+    bamm:description "The detailed description about the product type as enumeration."@en;
+    bamm:dataType xsd:string;
+    bamm-c:values ("manual transmission" "automated manual transmission" "automatic transmission" "continuously variable automatic transmissions (CVT transmission)").
+:Remanufacturing a bamm:Characteristic;
+    bamm:preferredName "Remanufacturing"@en;
+    bamm:description "Status of the transmission (first life or second life) and Remanufacturing- History."@en;
+    bamm:dataType :RemanufacturingStatus.
+:ServiceHistory a bamm-c:List;
+    bamm:preferredName "Service History"@en;
+    bamm:description "The dates of putting the transmission into service (Service History) as list."@en;
+    bamm:dataType xsd:dateTimeStamp.
+:ExpectedLifetime a bamm-c:Measurement;
```

Comment:
why not naming it Expected Lifespan as it is not measured in a timing unit

Reply:
done